### DISCUSSION TITLE
Welcome to TachiyomiS97 Discussions!

### DISCUSSION THREAD

**Discussion by Saud-97** [Saud-97/TachiyomiS97#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by sshJovian** [comment]
Bro, in the next update can you give an option to disable the "Toast notifications"?

**Reply by Saud-97** [reply]
Can you share which Toast notifications you are referring. Are they related to cloudflare bypasser?

**Reply by sshJovian** [reply]
Yes, specifically.

**Reply by Saud-97** [reply]
Can you open an issue so I can take a look on how to better solve this issue. 

Hiding them completely might cause issues since the app will pause network connections while it tries to bypass which might not be obvious and people might forget that they enabled this feature and wonder why is the app very slow. Especially that some sources just have harder cloudflare restrictions and the bypasser cannot work on them.
